src/Generator/Passes/CleanCommentsPass.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using CppSharp.AST;
+
+namespace CppSharp.Passes
+{
+    public class CleanCommentsPass : TranslationUnitPass
+    { 
+        public override bool VisitDeclaration(Declaration decl)
+        {
+            if (decl.Comment != null)
+            {
+                var summaryPara = (ParagraphComment)decl.Comment.FullComment.Blocks[0];
```

Comment:
@tritao As i said to @ddobrev  it was my attempt to make this work. Tomorrow i will ensure it will work by changing it from top to bottom.